learn-CSS-by-Building-a-blog/blog49.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Home | Learn Basic CSS by building a Blog</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css?family=Playfair+Display:700|Source+Sans+Pro:300,400&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div id="main">
+        <header>
+            <nav>
+                <ul class="nav-links">
+                    <li><a href="#">Intro</a></li>
+                    <li><a href="#">Blog</a></li>
+                </ul>
+                <a href="#" id="logo" >
+                    <img src="https://d33wubrfki0l68.cloudfront.net/e66392bc2941ad564b6956321e778f29073cf585/4190b/img/glyph-alternative.svg" alt="freeCodeCamp logo">
+                </a>
+            </nav>
+        </header>
+        <section class="container">
+            <div class="blog">
+                <div class="title">
+                    <h1>Blog</h1>
+                    <div class="divider"></div>
+                </div>
+                <div class="article">
+                    <div></div>
+                </div>
+            </div>
+        </section>
+    </div>
+</body>
+</html>
+
+<!-- 
+    Nest inside the `div` two `div` elements. The first `div` with the class of `title-date` and
```

Comment:
Inside the `div` you just added, nest two more `div` elements. Give the first `div` the class "title-date" and the second `div` the class "article-text".